scripts/transactions_exporter.py

```diff
@@ -17,22 +18,34 @@
 from yearn.prices import magic
 from yearn.yearn import Yearn
 
+sentry_sdk.set_tag('script','transactions_exporter')
+
 warnings.simplefilter("ignore", BrownieEnvironmentWarning)
 
 yearn = Yearn(load_strategies=False)
 
 logger = logging.getLogger('yearn.transactions_exporter')
 
-BATCH_SIZE = 5000
+BATCH_SIZE = {
+    Network.Mainnet: 5000,
+    Network.Fantom: 20000,
+}[chain.id]
```

Comment:
should we add a default for Arbitrum here ?

Reply:
sure but this will require me to test pull the arbitrum data to ensure our batch size is not too large or too small, I think this can be done in a separate PR

Reply:
okay, this will just hard crash here once this script is run with another chain so maybe init `BATCH_SIZE` with a default of e.g. 1000 and if chain.id is contained in the dict, overwrite it.

Reply:
That would have potential for infinite loop on lesser used chains, I'd prefer a hard failure until somebody can test a proper batch size, balancing between export time and frequency of vault txs, and add it to the dict.